pages/project/remove-user/index.js

```diff
@@ -0,0 +1,23 @@
+const { merge } = require('lodash');
+const { Router } = require('express');
+const bodyParser = require('body-parser');
+const content = require('./content');
+
+module.exports = () => {
+  const app = Router({ mergeParams: true });
+
+  app.post('/', bodyParser.urlencoded({ extended: true }), (req, res, next) => {
+    const { profileId } = req.body;
+    req.api(`/establishment/${req.establishmentId}/project/${req.project.id}/collaborators/${profileId}`, { method: 'DELETE' })
+      .then(() => next())
+      .catch(next);
+  });
+
+  app.post('/', (req, res, next) => {
+    res.locals.static.content = merge(res.locals.static.content, content);
+    req.notification({ key: 'success' });
+    res.redirect(req.query.referrer);
+  });
```

Comment:
Can we add a `GET` to this router that redirects to either the project page or edit collaborators page?

The reason being that if an error is thrown on the POST then there's a link to "try again" which reloads the same url on a GET. That should do _something_ other than 404.

Reply:
aah good plan